README.md

```diff
@@ -10,8 +10,8 @@ To use ExDoc in your Mix projects, first add ExDoc as a dependency:
 
 ```elixir
 def deps do
-  [{:earmark, "~> 0.1", only: :dev},
-   {:ex_doc, "~> 0.7", only: :dev}]
+  [{:earmark, "~> 0.1.17 or ~> 0.2", only: :dev},
```

Comment:
@josevalim @eksperimental So, this means that we can also update the [mix.exs](https://github.com/elixir-lang/ex_doc/blob/master/mix.exs#L14) file.